doc/using_pytorch.rst

```diff
@@ -303,7 +314,8 @@ It loads the model parameters from a ``model.pth`` file in the SageMaker model d
 
 However, if you are using PyTorch Elastic Inference, you do not have to provide a ``model_fn`` since the PyTorch serving
 container has a default one for you. But please note that if you are utilizing the default ``model_fn``, please save
-yor parameter file as ``model.pt`` instead of ``model.pth``. For more information on inference script, please refer to:
+your ScriptModule as ``model.pt``. If you are implementing your own ``model_fn``, please use TorchScript and ``torch.jit.save``
+to save your ScriptModule. For more information on inference script, please refer to:
```

Comment:
Add "then load it in your `model_fn` with `torch.jit.load`"

Reply:
added.